RiotSwiftUI/Modules/Template/SimpleScreenExample/View/TemplateSimpleScreen.swift

```diff
@@ -0,0 +1,111 @@
+// 
+// Copyright 2021 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import SwiftUI
+
+@available(iOS 14.0, *)
+struct TemplateSimpleScreen: View {
+
+    // MARK: - Properties
+    
+    // MARK: Private
+    
+    @Environment(\.theme) private var theme
+    @Environment(\.horizontalSizeClass) private var horizontalSizeClass
+    
+    private var horizontalPadding: CGFloat {
+        horizontalSizeClass == .regular ? 50 : 16
+    }
+    
+    // MARK: Public
+    
+    @ObservedObject var viewModel: TemplateSimpleScreenViewModel.Context
+    
+    /// The main content of the view to be shown in a scroll view.
+    var mainContent: some View {
```

Comment:
The way these properties are ordered feels strange to me. I generally prefer to have the dependees undearneath where they're used so that it all reads top to bottom.